src/app/public/style-loader.ts

```diff
@@ -14,15 +14,16 @@ export class SkyAppStyleLoader {
       return Promise.resolve();
     }
 
-    const fontAwesome = new FontFaceObserver('FontAwesome');
+    const fontAwesome4 = new FontFaceObserver('FontAwesome', { weight: 900 });
+    const fontAwesome5 = new FontFaceObserver('Font Awesome 5 Brands', { weight: 400 });
     const blackbaudSans = new FontFaceObserver('Blackbaud Sans');
 
     return Promise
       .all([
-        // Specify a character for FontAwesome since some browsers will fail to detect
-        // when the font is loaded unless a known character with a different width
-        // than the default is not specified.
-        fontAwesome.load('\uf0fc', SkyAppStyleLoader.LOAD_TIMEOUT),
+        // FA does not contain the latin characters "BESbwy", so we need a custom test string.
```

Comment:
What is "BESbwy"?

Reply:
Explained in the url immediately following the comment.